app/inbound/organisation/schema.js

```diff
@@ -0,0 +1,68 @@
+const Joi = require('joi')
+const { CALCULATION, ORGANISATION } = require('../../constants/types')
+
+const frnMin = 1000000000
+const frnMax = 9999999999
+const sbiMin = 105000000
+const sbiMax = 999999999
+const postcodeMax = 8
+
+module.exports = Joi.object({
+  sbi: Joi.number().integer().min(sbiMin).max(sbiMax).required().messages({
+    'number.base': 'sbi should be a type of number',
```

Comment:
I think technically better English here would be 'sbi should have type of number'